homeassistant/components/jellyfin/__init__.py

```diff
@@ -30,9 +30,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         user_id, connect_result = await validate_input(hass, dict(entry.data), client)
     except CannotConnect as ex:
         raise ConfigEntryNotReady("Cannot connect to Jellyfin server") from ex
-    except InvalidAuth:
-        LOGGER.error("Failed to login to Jellyfin server")
-        return False
+    except InvalidAuth as ex:
+        raise ConfigEntryAuthFailed(ex) from ex
```

Comment:
This should be a string, not the ex object.

Reply:
I believe this is exactly as the documentation describes: 
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#reauthentication